tests/apps/scout/test_scout_load_config.py

```diff
@@ -17,7 +18,16 @@ def test_validate_individual_display_name(sample_dict):
     assert ind_obj.sample_name == sample["sample_name"]
 
 
-def test_validate_mt_bam(sample_dict):
+@pytest.mark.parametrize(
```

Comment:
What abut importing the fixture_sample_dict and then converting it to a tuple in a new variable and supplying the tuple variable to the decorator here:
```
(
            "mt_bam",
            Path("path", "to", "reduced_mt.bam").as_posix(),
        ),
```

Reply:
That was a speedy review. Vincent helped sorting out the pytests in a later commit. I'll update this PR with some tests